charts/seed-bootstrap/charts/fluentd-es/values.yaml

```
@@ -30,3 +30,6 @@ fluentbit:
     role: logging
   ports:
     metrics: 2020
+  extensions:
+    filters:
```

```suggestion
    filters: ""
    parsers: ""
```

to make it more clear what type is expceted